include/class.ticket.php

```diff
@@ -1616,6 +1626,21 @@ function transfer($deptId, $comments, $alert = true) {
          return true;
     }
 
+    function claim() {
+        global $thisstaff;
+
+        if (!$thisstaff || !$this->isOpen() || $this->isAssigned())
+            return false;
+
+        $dept = $this->getDept();
+        if ($dept->assignMembersOnly() && !$dept->isMember($thisstaff))
+            return false;
+
+        $comments = sprintf(__('Ticket claimed by %s'), $thisstaff->getName());
```

Comment:
I have been using `_S` for these statements where the system primary language should be used instead of the current user's preference